README.md

```diff
@@ -49,9 +48,9 @@ To create new projects, use the `brigade-project` Helm chart. While inside the G
 repository cloned above, run these commands:
 
 ```console
-$ helm inspect values ./charts/brigade-project > myvalues.yaml
+$ helm inspect values brigade/brigade-project > myvalues.yaml
 $ # edit myvalues.yaml
-$ helm install --name my-project ./charts/brigade-project -f myvalues.yaml
+$ helm install --name my-project brigade/brigade-project -f myvalues.yaml
 ```
 
 When editing `myvalues.yaml`, follow the instructions in that file for configuring
```

Comment:
below here, _you do absolutely nothing in the file IF you're going to use the following script._. Why do we say this here? This should be in the "how to build something a bit more real" section, right?

Reply:
Re-organized